projects/developer/src/app/data/ingest/component.ts

```diff
@@ -3,6 +3,9 @@ import { Router, ActivatedRoute, Params } from '@angular/router';
 import { LazyLoadEvent, SelectItem } from 'primeng/primeng';
 import { MessageService } from 'primeng/components/common/messageservice';
 import { BreakpointObserver, BreakpointState } from '@angular/cdk/layout';
+import { Observable } from 'rxjs';

```

Comment:
all of the changes to this file shouldn't be in this PR, I think you may have incorrectly merged from upstream